ml-algorithms/src/test/java/org/opensearch/ml/engine/algorithms/remote/HttpJsonConnectorExecutorTest.java

```diff
@@ -220,7 +220,7 @@ public void invokeRemoteModel_nullHttpClient_throwMLException() throws NoSuchFie
                 new ExecutionContext(0, new CountDownLatch(1), new AtomicReference<>()),
                 actionListener
             );
-        ArgumentCaptor<Exception> argumentCaptor = ArgumentCaptor.forClass(Exception.class);
+        ArgumentCaptor<Exception> argumentCaptor = ArgumentCaptor.forClass(NullPointerException.class);
```

Comment:
Why change to NullPointerException?

Reply:
This will fail the UT

Reply:
I don't know why, it's just failed, https://github.com/opensearch-project/ml-commons/actions/runs/8915168818/job/24484230861

Reply:
Seems this UT is flaky, I reverted the change.